docs/release_notes/upgrading_to_destinations_v2.md

```diff
@@ -71,7 +71,7 @@ Versions are tied to the destination. When you update the destination, **all con
 - [Rolling back to Legacy Normalization](#oss-only-rolling-back-to-legacy-normalization)
 
 :::info
-If you were a Destinations V2 "Early Access" user, you will still need to opt-into the latest connector version.  However, you will not experince the data migration and syncs will contuie to work as they have been since the "early access" period began. 
+If you were a Destinations V2 "Early Access" user, you will still need to opt-into the latest connector version. However, you will not experince the data migration and syncs will contuie to work as they have been since the "early access" period began.
```

Comment:
```suggestion
If you were a Destinations V2 "Early Access" user, you will still need to opt-into the latest connector version. However, you will not experience the data migration and syncs will continue to work as they have been since the "early access" period began.
```